### ISSUE TITLE
startup failure when using centralized user management

### ISSUE THREAD

**Issue by grturner** [teejee2008/polo#165]
polo fails to initialize itself correctly when users are centrally managed. I'm fairly sure that it has to do with how polo obtains usernames and home directories which is to read in lines from /etc/passwd and in the case that the user is not found in the passwd file, it defaults to using the / directory for creating the dotfiles. 

In my case, user management is done centrally by a FreeIPA instance and users are not found locally in passwd. Perhaps a better approach would be to use getent such as `getent passwd $USER` as a fallback if the user is not found in passwd. 

```
[grturner@fedora ~]$ polo-gtk 
E: File not found: /.config/user-dirs.dirs
Architecture: 64-bit
Host Name: fedora.ws02
CPU Cores: 8
RAM: 15761 MB
E: .config/polo
E: Operation not supported
E: 
E: .config/polo/open
E: Operation not supported
E: 
E: .config/polo/remotes
E: Operation not supported
E: 
E: .config/polo/rclone-mounts
E: Operation not supported
E: 
E: .config/polo/rclone-cache
E: Operation not supported
E: 
E: Operation not supported
E: Operation not supported
E: Operation not supported
E: Operation not supported
E: .config/fish/functions
E: Operation not supported
E: 
E: Error opening file “/home/grturner/.config/fish/functions/fish_prompt.fish”: No such file or directory
E: Failed to copy file: '/usr/share/polo/files/fish_prompt.fish', '.config/fish/functions/fish_prompt.fish'
E: 
E: chown: missing operand after ‘.config/fish/functions/fish_prompt.fish’
Try 'chown --help' for more information.

E: 
E: chmod: cannot access '.config/fish/functions/fish_prompt.fish': No such file or directory

E: Error opening file “/home/grturner/.config/polo/bashrc”: No such file or directory
E: Failed to copy file: '/usr/share/polo/files/bashrc', '.config/polo/bashrc'
E: 
E: chown: missing operand after ‘.config/polo/bashrc’
Try 'chown --help' for more information.

E: 
E: chmod: cannot access '.config/polo/bashrc': No such file or directory

E: File not found: /.config/gtk-3.0/bookmarks
GTK3 Version: 3.22
pwd: /usr/share/polo/files/gtk-theme
```

**Comment by grturner** [comment]
See #166 for a proposed fix.

**Comment by teejee2008** [comment]
Fixed in v18.2. Let me know if you face any issues.
